packages/cli/src/check.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpService } from "./http";
+import { loadLocalConfig } from "./util";
+
+export async function check({
```

Comment:
check? check what :) the weather? ^^ there is no limit in naming functions..
even after looking into it its not clear what is going on here.. what is being checked. what is invalid in the return? the local one, the remote one?

Reply:
Good catch!